tests/test_tokenization_common.py

```diff
@@ -712,3 +733,147 @@ def _check_no_pad_token_padding(self, tokenizer, sequences):
 
             # add pad_token_id to pass subsequent tests
             tokenizer.add_special_tokens({"pad_token": "<PAD>"})
+
+    @require_torch
+    def test_torch_encode_plus_sent_to_model(self):
+        from transformers import MODEL_MAPPING, TOKENIZER_MAPPING
+
+        MODEL_TOKENIZER_MAPPING = merge_model_tokenizer_mappings(MODEL_MAPPING, TOKENIZER_MAPPING)
+
+        tokenizer = self.get_tokenizer()
+
+        if tokenizer.__class__ not in MODEL_TOKENIZER_MAPPING:
+            return
+
+        config_class, model_class = MODEL_TOKENIZER_MAPPING[tokenizer.__class__]
+        config = config_class()
+
+        if config.is_encoder_decoder or config.pad_token_id is None:
```

Comment:
Did it break for seq2seq? Happy to help

Reply:
I believe there was no issue with BART, but there were some with T5. Namely the `decoder_input_ids` were a necessary input for the model, which is not generated for sequence to sequence models when using `encode_plus` or `batch_encode_plus`.

I think we would have to clear the expected behavior of these methods for seq2seq models before testing them.